lib/backend/k8s/k8s_fv_test.go

```diff
@@ -209,6 +209,10 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 		go cb.ProcessUpdates()
 	})
 
+	FIt("should allow watches to timeout", func() {
```

Comment:
Or we can just remove this test and do unit testing or system testing of the timeout logic